articles/extensions/logstash/v2/index.md

```diff
@@ -0,0 +1,65 @@
+| **BATCH_SIZE** | The ammount of logs to be read on each execution. Maximun, and default, is `100`. |
+| **LOGSTASH_URL** <br/><span class="label label-danger">Required</span> | Your Logstash URL as defined for use with `logstash-input-http` plugin. |
+| **LOGSTASH_INDEX** <br/><span class="label label-danger">Required</span> | Your Logstash Index to which the logs will be routed. |
+| **LOGSTASH_TOKEN** | The token required for your Logstash deployments that will be included in the querystring. |
+| **LOGSTASH_USER** | The Logstash user. |
+| **LOGSTASH_PASSWORD** | The password associated with your Logstash user. |
+| **START_FROM** | The checkpoint ID of the log from where you want to start. |
+| **SLACK_INCOMING_WEBHOOK** | The Slack incoming webhook URL used to send relevant updates. |
+| **SLACK_SEND_SUCCESS** | Toggle for sending verbose notifications to Slack. |
+| **LOG_LEVEL** | The minimal log level of events that you would like sent to Logstash. |
+| **LOG_TYPES** | The events for which logs should be exported. |
+
+Once you have provided this information, click the _Install_ button to finish installing the extension.
+
+## Use the Extension
+
+To view all scheduled jobs, navigate to the [Extensions](${manage_url}/#/extensions) page of the [Management Portal](${manage_url}), click on the **Installed Extensions** link, and select the **Auth0 Logs to Logstash** line. 
+
+There you can see the job you just created, modify its state by toggling the **State** switch, see when the next run is due and what was the result of the last execution. 
+
+![View Cron Jobs](/media/articles/extensions/logstash/view-cron-jobs.png)
+
+You can view more details by clicking on the job you created. In this page you can view details for each execution, reschedule, access realtime logs, and more.
+
+![View Cron Job Details](/media/articles/extensions/logstash/view-cron-details.png)
+
+## Replay Logs
+
+In the event of a Logstash failure or service interuption you can replay the logs starting from the failed log.
+
+To replay logs: 
+
+1. Get the checkpoint ID of the failed log.
+2. Go to the Auth0 Logs to Logstash extension settings.
+3. Enter the checkpoint in the `START_FROM` field.
+4. Click the __Save__ button to replay the failed logs.
+
+## Slack Integration
+
+To set up [Slack](https://slack.com/) integration, provide an [Incoming Webhook URL](https://api.slack.com/incoming-webhooks) to the `SLACK_INCOMING_WEBHOOK` field in the Auth0 Logs to Logstash [extension settings](${manage_url}/#/extensions).
+
+![Slack Settings](/media/articles/extensions/logstash/slack-settings.png)
+
+The extension sends failed transaction notifications to Slack with the checkpoint code displayed in the message. You can also enable verbose notifications by turning on the `SLACK_SEND_SUCCESS` setting.
+
+![Slack Message](/media/articles/extensions/logstash/slack-message.png)
```

Comment:
Add a border to this image (see https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#borders)